### PR TITLE
React refactor

### PR DESCRIPTION
**Pull request by Stanleeeeee**
In this project part, I refactored the Calculator component I developed. Instead of using class-based components, I used hooks.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
+import React, { useState } from 'react';
+import Button from '../components/Button.js';
```
Comment (Ismailco) [inline src/components/calculator.js]: - [ ] This component produces an error,  please consider changing the name of the component to `Button.js` or changing this line to `import Button from '../components/button.js';` to fix this problem

Screenshot_20220611_170654

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Stanleeeeee,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Stanleeeeee — COMMENTED** [review]
Review Correction about react refactor using hooks

File: src/components/button.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
```
Comment (roshan-bajgain) [inline src/components/button.js]: - [ ] Kindly rename this `button.js` file with the `Button.js` file as it is causing the error. So kindly make changes and ask for another reviewer. Everything is fine besides this issue.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Stanleeeeee, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Stanleeeeee — COMMENTED** [review]
I renamed the button.js in the calculator file

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Stanleeeeee  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**